src/main.ts

```diff
@@ -1,22 +1,24 @@
 import * as core from '@actions/core'
 import * as github from '@actions/github'
-import { BACKEND_BASE_URLS_BY_ENV } from './constants'
+import { BASE_URLS_BY_ENV } from './constants'
 import { waitTillWorkflowExecuted } from './wait-till-workflow-executed'
 import { TokenData } from './types'
 
 export async function run(): Promise<void> {
   try {
     const workflowId: string = core.getInput('workflowId')
     const chassyRefreshTokenB64 = process.env.CHASSY_TOKEN
-    if (!chassyRefreshTokenB64) {
+    if (chassyRefreshTokenB64 === undefined) {
       throw new Error('CHASSY_TOKEN not provided in environment')
+    } else if (chassyRefreshTokenB64 === '') {
+      throw new Error('CHASSY_TOKEN value is empty string')
     }
```

Comment:
Why do we need to check for an empty string?
how is it different from case with "CHASSY_TOKEN not provided in environment"?

Reply:
Well in my debugging, I was setting the variable equal to a variable in secrets, but because the variable in secrets was undefined, it was getting an empty string. I wasn't sure if my issue was that my yaml was malformed or that my variable was empty. I suppose this change just points you in the right direction for debugging quicker. In my case, I was obviously setting the variable, but the variable I was setting it to happened to be empty rather than having not provided the variable at all. 

I can change it back if it's preferred, but I think the more specificity in error messages the better for code users will interact with directly.

Reply:
let's keep it, I was just wondering about the case this message can be needed